src/dataconnect/prompts.ts

```diff
@@ -21,3 +21,10 @@ export async function promptDeleteConnector(
     utils.logLabeledSuccess("dataconnect", `Connector ${connectorName} deleted`);
   }
 }
+
+export function locationInFirebaseJsonWarning() {
```

Comment:
Is this used? I saw the same string in `src/config.ts`

Reply:
Oops, this was accidentally left in, removing.